docs/cluster-configuration.md

```diff
@@ -130,26 +130,52 @@ or new features that are not enabled by default.
 To enable a feature, add its name to the `featureGates` list and set it to `true`. Missing or `false` feature gates
 disables the feature.
 
+Some of the feature gates are used to set feature gates in the KubeVirt CR. This feature gates are described in the
+[KubeVirt feature](#kubevirt-feature-gates) gates sections.
+
+### withHostPassthroughCPU Feature Gate
+This feature gate is deprecated and is ignored.
+
+### deployKubeSecondaryDNS Feature Gate
+Set the `deployKubeSecondaryDNS` feature gate to true to allow deploying KubeSecondaryDNS by CNAO.
+For additional information, see here: [KubeSecondaryDNS](https://github.com/kubevirt/kubesecondarydns)
+
+**Default**: `false`
+
+### Feature Gates Example
+
+```yaml
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  infra: {}
+  workloads: {}
+  featureGates:
+    alignCPUs: true
+    deployKubeSecondaryDNS: true
+```
+
+## KubeVirt feature gates
+The following feature gates are used to set feature gates in the KubeVirt CR. For few of them, HCO will perform some
+more functionality, beside just setting the feature gate in the KubeVirt CR.
```

Comment:
`some more functionality` is a bit cryptic

Reply:
Removed